src/settings/BehaviorForm.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  injectIntl,
+  intlShape,
+  FormattedMessage,
+} from 'react-intl';
+import {
+  Form,
+} from 'react-final-form';
+
+import {
+  Pane,
+  PaneFooter,
+  Select,
+  Button,
+} from '@folio/stripes/components';
+
+import RowComponent from './RowComponent';
+
+const deletedRecordsSupportSelectValues = (formatMessage) => [
+  {
+    value: 'no',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.deletedRecordsSupport.no' }),
+  },
+  {
+    value: 'transient',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.deletedRecordsSupport.transient' }),
+  },
+  {
+    value: 'persistent',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.deletedRecordsSupport.persistent' }),
+  },
+];
+
+const suppressedRecordsProcessingSelectValues = (formatMessage) => [
+  {
+    value: true,
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.suppressedRecordsProcessing.true' }),
+  },
+  {
+    value: false,
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.suppressedRecordsProcessing.false' }),
+  },
+];
+
+const errorsProcessingSelectValues = (formatMessage) => [
+  {
+    value: '200',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.errorsProcessing.200' }),
+  },
+  {
+    value: 'error',
+    label: formatMessage({ id: 'ui-oai-pmh.settings.behavior.errorsProcessing.error' }),
+  },
+];
+
+class BehaviorForm extends Component {
+  static propTypes = {
+    label: PropTypes.node.isRequired,
+    intl: intlShape.isRequired,
+  };
+
+  renderFooter = () => {
+    return (
+      <PaneFooter
+        renderEnd={(
+          <Button
+            data-test-behavior-form-button-save
+            type="submit"
+            buttonStyle="primary paneHeaderNewButton"
+            marginBottom0
+          >
+            <FormattedMessage id="stripes-core.button.save" />
+          </Button>
+        )}
+      />
+    );
+  };
+
+  render() {
+    const {
+      label,
+      intl: {
```

Comment:
I would suggest using [`<IntlConcumer />`](https://github.com/folio-org/stripes-core/tree/master/src/components/IntlConsumer) component whenever possible instead of this approach. This way it won't be needed to deal with `injectIntl` HOC. Additionally, this approach is considered preferable in the community.

Reply:
Fixed